DOCS-674 - Static log monitor frequency evaluation (#5036)
SumoLogic/sumologic-documentation: docs/alerts/monitors/create-monitor.md

Triggers are evaluated by balancing the requirement of timely alert notifications while ensuring that monitor data is indeed available to evaluate trigger conditions.

* For [static logs monitors](#static-detection-method), you can control trigger monitor evaluation frequency using the options below. If `Alert when result is <greater/less> than <_> within <X>. Evaluate trigger every <Y>.`:
   | When detection window (X) is | Evaluate trigger every (Y) |
   |:-----|:-----------------------|

>    | 2m   | 1m, 2m  |
>    | 5m   | 1m, 2m, 5m |
>    | 10m  | 1m, 2m, 5m, 10m |
>    | 15m  | 1m, 2m, 5m, 10m |
>    | 30m  | 2m, 5m, 10m, 20m |
>    | 1h   | 2m, 5m, 10m, 20m |
>    | 3h   | 10m, 20m, 40m, 1h |
>    | 6h   | 10m, 20m, 40m, 1h |
>    | 12h  | 20m, 40m, 1h |
>    | 24h  | 20m, 40m, 1h |
>    | 7d   | 20m, 40m, 1h |

Reviewer: ```suggestion
   | 5m   | 1m, 2m |
   | 10m  | 1m, 2m, 5m |
   | 15m  | 1m, 2m, 5m, 10m |
   | 30m  | 2m, 5m, 10m, 20m |
   | 1h   | 2m, 5m, 10m, 20m |
   | 3h   | 10m, 20m, 40m, 1h |
   | 6h   | 10m, 20m, 40m, 1h |
   | 12h  | 20m, 40m, 1h |
   | 24h  | 20m, 40m, 1h |
```
- Adding some minor corrections
- Removing `2m` and `7d` detection window because those are exceptions that we made only for a few customers.

When configuring monitor trigger conditions, you can set a resolution window to resolve alerts quickly once the underlying issue is fixed. The resolution window specifies how long a monitor will wait before resolving an alert after the issue is corrected.

For example, if your monitor evaluates the last 1 hour, you can set a resolution window of 15 minutes. Once the resolution window is continuously satisfied for 15 minutes, the alert will resolve automatically.<br/><img src={useBaseUrl('img/alerts/monitors/config-resolution-window-logs.png')} alt="config-resolution-window" style={{border: '1px solid gray'}} width="700"/>

#### Static detection method

**Example: Logs - Static - Critical and Warning**  

<img src={useBaseUrl('img/alerts/monitors/logs-trigger-type.png')} alt="logs trigger type.png" style={{border: '1px solid gray'}} width="600"/>

> `Alert when result is <threshold type> <threshold> within <time range - trigger>. Evaluate every <time range - frequency>.`

Author: ```suggestion
`Alert when result is <threshold type> <threshold> within <time range - trigger>. Evaluate every <trigger - frequency>.`
```

When configuring monitor trigger conditions, you can set a resolution window to resolve alerts quickly once the underlying issue is fixed. The resolution window specifies how long a monitor will wait before resolving an alert after the issue is corrected.

> For example, if your monitor evaluates the last 1 Hour, you can set a resolution window of 15 Minutes. Once the resolution window is continuously satisfied for 15 Minutes, the alert will resolve automatically.<br/><img src={useBaseUrl('img/alerts/monitors/config-resolution-window-logs.png')} alt="config-resolution-window" style={{border: '1px solid gray'}} width="700"/>

Author: ```suggestion
For example, if your monitor evaluates the last 1 hour, you can set a resolution window of 15 minutes. Once the resolution window is continuously satisfied for 15 minutes, the alert will resolve automatically.<br/><img src={useBaseUrl('img/alerts/monitors/config-resolution-window-logs.png')} alt="config-resolution-window" style={{border: '1px solid gray'}} width="700"/>
```

#### Static detection method

**Example: Logs - Static - Critical and Warning**  

<img src={useBaseUrl('img/alerts/monitors/logs-trigger-type.png')} alt="logs trigger type.png" style={{border: '1px solid gray'}} width="600"/>

`Alert when result is <threshold type> <threshold> within <time range - trigger>. Evaluate every <time range - frequency>.`

| Parameter | Description |
|:--|:--|
| `<threshold type>` | How you want the value compared. Select **greater than**, **greater than or equal**, **less than or equal**, or **less than**. |
| `<threshold>` | The value against which the trigger will be evaluated. You can specify any valid numeric value up to **1,000**. |
| `<time range - trigger>` | The duration of time to evaluate. Values range from 2 Minutes to 24 Hours (or 7 Days, by request only). |

> | `<time range - frequency>` | The frequency that the monitor is evaluated. |

Reviewer: nit: this should be `The frequency that the trigger is evaluated.` because each trigger has its own frequency

> | `<time range - frequency>` | The frequency that the monitor is evaluated. |

Reviewer: ```suggestion
| `<trigger - frequency>` | The frequency that the trigger is evaluated. |
```
adding this suggestion because each trigger has its own frequency

<img src={useBaseUrl('img/alerts/monitors/logs-static-missing.png')} alt="logs-static-missing" style={{border: '1px solid gray'}} width="600" />

> `Alert when missing data within <time range>. Evaluate every <time range - frequency>.`

Author: ```suggestion
`Alert when missing data within <time range - trigger>. Evaluate every <trigger - frequency>.`
```

<img src={useBaseUrl('img/alerts/monitors/logs-static-missing.png')} alt="logs-static-missing" style={{border: '1px solid gray'}} width="600" />

`Alert when missing data within <time range>. Evaluate every <time range - frequency>.`

| Parameter | Description |
|:--|:--|
| `<time range - trigger>` | The duration of time to evaluate (values range from 5 minutes to 24 hours). |

> | `<time range - frequency>` | The frequency that the monitor is evaluated. |

Reviewer: ```suggestion
| `<trigger - frequency>` | The frequency that the trigger is evaluated. |
```
adding this suggestion because each trigger has its own frequency
